site/src/components/Navbar/AccountDropdown.tsx

```diff
@@ -0,0 +1,102 @@
+import { VFC, useState, useContext, useRef } from "react";
+import {
+  Box,
+  Typography,
+  ListItemButton,
+  Divider,
+  Popover,
+} from "@mui/material";
+import { Link } from "../Link";
+import { Button } from "../Button";
+import UserContext from "../context/UserContext";
+import { apiClient } from "../../lib/apiClient";
+import { UserAvatar } from "../UserAvatar";
+
+export const AccountDropdown: VFC = () => {
+  const { user, refetch } = useContext(UserContext);
+
+  const buttonRef = useRef<HTMLButtonElement>(null);
+
+  const [open, setOpen] = useState<boolean>(false);
+
+  const handleClick = () => {
+    setOpen(true);
+  };
+
+  const handleClose = () => {
+    setOpen(false);
+  };
+
+  const handleLogout = async () => {
+    await apiClient.post("logout");
+
+    refetch();
```

Comment:
It might be better to have a 'clear' method for the user - it saves the network request, and achieves what we know we want to do (clear the user from state) in a more direct way.

Reply:
I'll do the same for the `handleLogin` function (directly update the user context using a `setUser` dispatch method using the response of the `/api/loginWithLoginCode` request)

Reply:
Have done this here: https://github.com/blockprotocol/blockprotocol/pull/84/commits/264e2a914e89b89d4882eb71a894fb97075f8f79